documentation/docs/30-advanced/10-advanced-routing.md

```diff
@@ -227,9 +225,41 @@ src/routes/
 └ +layout.svelte
 ```
 
+#### Breaking out of the root layout
+
+There is no way to break out of the root layout. You can be sure that it's always present in your app and for example put app-wide UI or behavior in it - and if you don't define one, it falls back to a simple `<slot />` component. If you want some pages to _not_ have the same layout as the rest, then you have two options:
```

Comment:
We need to phrase this differently — it's framed as a limitation rather than a deliberate design choice, and it's weird to have a 'how to do X' section that begins 'you can't'

```suggestion
#### Ignoring all layouts

The root layout applies to every page of your app. If you want some pages to _not_ have the same layout as the rest, then you have two options:
```

Reply:
I phrased it specifically that way because on Discord people phrase their questions more around "breaking out of" than "ignoring" - but I agree that it's somewhat clunky the way I've written it.